test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,22 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def get_metadata_max_size(cache_size: Size, cache_line_size: Size, number_of_cores=4096):
+        """Get memory size according to requirement """
+        constant = int(Size(100, Unit.MebiByte))
+        cache_size_value = int(cache_size.value)
```

Comment:
I suggest using cache_size.get_value(Unit.MebiByte) to make sure that everything is counted with proper unit. Please use this method everywhere where you are getting the size's value